compiler/next/include/chpl/uast/MultiDecl.h

```diff
@@ -48,19 +47,18 @@ namespace uast {
   list of VariableDecls.  Note that the initial value and/or type is inferred
   from later declarations.
 
-  Since the MultiDecl does not declare a Sym itself, it is not a
-  subclass of Decl. Rather, it contains several VariableDecls.
+  Since the MultiDecl does not itself have a name, it is not
+  a NamedDecl. Rather, it can contain NamedDecls.
```

Comment:
I suggest you move the first of these sentences to the previous paragraph and drop the second sentence. The second sentence duplicates what is already written above, with less precision.

Also please extend "MultiDecl containing a list of VariableDecls" to include Tuples and Comments.